pkg/registry/manifests_test.go

```diff
@@ -70,7 +78,7 @@ func TestFindImagesInManifest(t *testing.T) {
 	manifestData, err := readManifest(manifestPath)
```

Comment:
Additional note to the suggestion above. This only works because we're dealing with `any` type. The `GetAllImages` function iterates over the manifests when calling `storeManifestImageNames` whereas this one does not:

```
// GetAllImages(...)
manifests, err := readManifest(manifestPath)
if err != nil {
	return nil, fmt.Errorf("error reading manifest %w", err)
}

for _, manifestData := range manifests {
	storeManifestImageNames(manifestData, extractedImagesSet)
}
```